db/migrate/20220217005515_create_form1095_bs.rb

```diff
@@ -0,0 +1,28 @@
+class CreateForm1095Bs < ActiveRecord::Migration[6.1]
+  def change
+    create_table :form1095_bs, id: :uuid do |t|
+      t.string :veteran_icn, null: false
+
+      t.string :first_name, null: false
+      t.string :middle_name
+      t.string :last_name, null: false
+      t.date :birth_date
+      t.string :ssn
+      t.string :address, null: false
+      t.string :city
+      t.string :state
+      t.string :country
+      t.string :zip_code
+      t.string :province
+      t.string :foreign_zip
```

Comment:
Please try to avoid storing PII in the database. There are a few options:
- Store in Redis. Store the data in a single value field and parse it upon retrieval. (this would be faster too)
- If you must do this in the DB, you MUST cyphertext all PII.
- Reduce PII to only what you actually need in the DB (encrypted) and leave the rest in redis. We already store most of this information in a redis object.